README.md

```diff
@@ -4,13 +4,29 @@ This is a command-line tool to take the [FedRAMP](http://www.fedramp.gov/) Syste
 
 ## Usage
 
-Requires [Go](https://golang.org/) 1.6+.
+Requires [Go](https://golang.org/) 1.6+. 
+
 
 1. Install the tool:
 
+    For Ubuntu/Debian:
     ```bash
+    sudo apt-get install libxml2-dev 
     go get github.com/opencontrol/fedramp-templater
     ```
+    
+    For OsX:
+    ```
+    brew install homebrew/dupes/apple-gcc42
+    export CC=/usr/local/Cellar/apple-gcc42/4.2.1-5666.3/bin/gcc-4.2
+    brew install libxml2
+    ln -s /usr/local/Cellar/libxml2/2.9.4/lib/pkgconfig/libxml-2.0.pc /usr/local/lib/pkgconfig/libxml-2.0.pc
```

Comment:
No they don't work (thanks for writing that though).  I went with the notes at https://github.com/moovweb/gokogiri/issues/14. 

Maybe the README should be as simple as possible, but note 

> Some users report problems with the installation of the required `gokogiri` package. We don't have a reliable resolution, so please see (link to some issue #) for tips and to ask for help

P.